### DISCUSSION TITLE
Smooth the trajectory

### DISCUSSION THREAD

**Discussion by Entongsu** [NVlabs/curobo#244] [Software Q&A] [answered]
Hi,

I wonder whether there is any way to make the trajectory smoother. I have found that sometimes, the robot's trajectory is not smooth enough. Thank you!

**Comment by balakumar-s** [comment, marked as answer]
Increasing `finetune_trajopt_iters` will generate smoother trajectories. A good starting value would be 500. If increasing this value does not help, you can reduce `trajopt_dt` to 0.1 and see if that helps. Both of these values are in https://curobo.org/_api/curobo.wrap.reacher.motion_gen.html#curobo.wrap.reacher.motion_gen.MotionGenConfig.load_from_robot_config 

If you are observing non-smooth trajectories when running at a `velocity_scale<1.0`, we pushed a recent release where you can use `time_dilation_factor` to slow down trajectories instead of changing `velocity_scale`.
